conf/config.inc.php

```diff
@@ -288,3 +293,8 @@
 
 # Launch a posthook script after successful password change
 #$posthook = "/usr/share/self-service-password/posthook.sh";
+
+# Allow to override current settings with local configuration
+if (file_exists (dirname (__FILE__) . '/config.inc.local.php')) {
```

Comment:
dirname ( __ FILE __) is old syntax, __ DIR __ should be used (we are no longer supporting php 5.3 and older)